src/utils/index.js

```diff
@@ -0,0 +1,53 @@
+
+export const mapComment = commentLen => {
+  if (commentLen > 2) {
+    return `${commentLen} comments`;
+  } else if (commentLen === 1) {
+    return `${commentLen} comment`;
+  } else {
+    return `discuss`;
+  }
+};
+
+export function getSourceUrl(url) {
+  const regex = /^(?:https?:\/\/)?(?:[^@\n]+@)?(?:www\.)?([^:/\n]+)/;
+  let match = regex.exec(url);
+
+  if (!match) return "";
+
+  let sourceUrl = match[1] ? match[1] : "#";
```

Comment:
Also not quite sure about the two error states of returning `""` and `"#"`. Maybe the second one is on purpose? This is hard to follow, comments would be appreciated.

Reply:
updated